[Privacy Sandbox] Glossary: WebID -> FedCM (#1717)
GoogleChrome/developer.chrome.com: site/en/docs/privacy-sandbox/glossary/index.md

Australia doesn't reduce entropy if you already know the person is from
Kangaroo Island.

> ## Federated identity, federated login {: #federated-identity }

Reviewer: ```suggestion
## Federated identity (federated login) {: #federated-identity }
```

Reviewer: How about this formatting if they are synonyms? Feels like it could be two things when it's comma separated. Just an idea, no strong opinion on this.

Author: I feel good about. I'm going to review the rest of the glossary to ensure it's the same everywhere.

Reviewer: Hehe love having some good word nerd collaboration

A third-party platform to allow a user to sign in to a website, without
requiring the site to implement their own identity service.

> ## Federated Credentials Management (FedCM) {: #fedcm }

Reviewer 2: I noticed between https://wicg.github.io/FedCM/ and https://github.com/wicg/fedcm, "Federated Credentials Management" and "Federated Credential Management API" are used interchangeably. I'm asking the team which is preferred.

> ## Federated Credentials Management (FedCM) {: #fedcm }

Reviewer 2: In the spec repository and within the team "Federated Credentials Management" and "Federated Credential Management API" are interchangeably used in different places, but I'll just follow what's written in the spec.
```suggestion
## Federated Credential Management API (FedCM) {: #fedcm }
```

> Federated Credentials Management is a proposal for a privacy-preserving

Reviewer 2: ```suggestion
Federated Credential Management API is a proposal for a privacy-preserving
```
